tasks/kustomization.yaml

```diff
@@ -11,4 +11,6 @@ resources:
 - buildah.yaml
 - git-clone.yaml
 - s2i-nodejs.yaml
-- s2i-java.yaml
\ No newline at end of file
+- s2i-java.yaml
+- cosign-verify.yaml
```

Comment:
I was going to suggest calling it cosign-verify-image-sig for understandably, but actually it's better to stick with the convention established by cosign's own commands.